docs/extend/extend-apiml/zowe-api-mediation-layer-security-overview.md

```diff
@@ -33,6 +33,18 @@ API ML uses the following authentication methods:
 - **TLS client certificates**
     - Certificates are used for service-only requests
 
+- **OIDC authentication**
+ 
+  API ML is now able to authenticate mainframe users with external/distributed IDP (OIDC provider), such as OKTA, KeyCloak and others.
+  Client applications can ask their users to log in at OIDC provider's authentication page, and then access APIs with the JWT provided by the external IDP. The following process outlines that basic flow:
```

Comment:
same question about whether it's IAM / IDP, are these interchangeable?

Reply:
IAM was removed from the article, as we don't prereq full-blown Identity and Access Management. It is enough to have  OAuth2 Identity Provider, preferably with OIDC authentication layer capabilities. In the article, I use intensively OIDC instead of OAuth2. This probably deserves a revision to decide if to change it OIDC word for OAuth2 and state it precisely.